TidepoolKit/Third Party/Reachability.swift

```diff
@@ -0,0 +1,299 @@
+/*
```

Comment:
The only real use for Reachability APIs is when deciding whether to retry a network connection, or to adjust error messaging.  Neither of those seem to be the uses here.  From what I can see, it's being used to prevent requests from being made, and prevent the user from engaging in login or other network activities. Both of those will prevent desired actions, when Reachability returns a false negative (network is powered down, but a request would trigger it to come up and would be successful).

Reply:
I'd recommend we remove all the reachability code from this framework.